mmv1/third_party/terraform/services/privilegedaccessmanager/resource_privileged_access_manager_entitlement_test.go.erb

```diff
@@ -0,0 +1,137 @@
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderBetaFactories(t),
+		CheckDestroy:             testAccCheckPrivilegedAccessManagerEntitlementDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccPrivilegedAccessManagerEntitlement_privilegedAccessManagerEntitlementBasicExample_basic(context),
+			},
+			{
+				ResourceName:            "google_privileged_access_manager_entitlement.tfentitlement",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"location", "entitlement_id", "parent"},
+			},
+			{
+				Config: testAccPrivilegedAccessManagerEntitlement_privilegedAccessManagerEntitlementBasicExample_update(context),
+			},
+			{
+				ResourceName:            "google_privileged_access_manager_entitlement.tfentitlement",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"location", "entitlement_id", "parent"},
+			},
+		},
+	})
+}
+
+func testAccPrivilegedAccessManagerEntitlement_privilegedAccessManagerEntitlementBasicExample_basic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_privileged_access_manager_entitlement" "tfentitlement" {
+    provider =  google-beta
+    entitlement_id = "tf-test-example-entitlement%{random_suffix}"
+    location = "global"
+    max_request_duration = "43200s"
+    parent = "projects/%{project_name}"
+    requester_justification_config { 
+      unstructured{}
+    }
+    eligible_users {
+      principals = ["user:gkmr@google.com"]
+    }
+    privileged_access{
+      gcp_iam_access{
+        role_bindings{
+          role = "roles/storage.admin"
+          condition_expression = ""
+        }
+        resource = "//cloudresourcemanager.googleapis.com/projects/%{project_name}"
+        resource_type = "cloudresourcemanager.googleapis.com/Project"
+      }
+    }
+    additional_notification_targets {
+      admin_email_recipients     = ["user@example.com"]
+      requester_email_recipients = ["user@example.com"]
+    }
+    approval_workflow {
+    manual_approvals {
+      require_approver_justification = true
+      steps {
+        approvals_needed          = 1
+          approver_email_recipients = ["user@example.com"]
+          approvers {
+            principals = ["user:itsvarsharma@google.com"]
+        }
+      }
+    }
+  }
+}
+`, context)
+}
+
+func testAccPrivilegedAccessManagerEntitlement_privilegedAccessManagerEntitlementBasicExample_update(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_privileged_access_manager_entitlement" "tfentitlement" {
+    provider =  google-beta
+    entitlement_id = "tf-test-example-entitlement%{random_suffix}"
+    location = "global"
+    max_request_duration = "4300s"
+    parent = "projects/%{project_name}"
+    requester_justification_config {    
+      not_mandatory{}
+    }
+    eligible_users {
+      principals = ["user:itsvarsharma@google.com]
```

Comment:
```suggestion
      principals = ["user:itsvarsharma@google.com"]
```
missing closing quote, but also these references to your personal emails need to be updated to something generic if possible. it doesn't seem like these are validated (as neither of these emails would have valid permissions on the CI project), so something like "a@example.com" or similar if that would work.